infrastructure/docker/cover-service-landing/etc/confd/templates/default.conf.tmpl

```diff
@@ -8,6 +8,12 @@ server {
         index index.htm index.html;
     }
 
+    location = /robots.txt {
+       add_header Content-Type text/plain;
+       add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
```

Comment:
This will only add the `X-Robots-Tag`response header for calls to `robots-txt`. As i understand the docs we need to return that header on any call to any route we don't want indexed

Reply:
hmmm

Reply:
Added the header to the other locations...